### PR TITLE
added example issue number to kitematic contributors guide instead of…

### PR DESCRIPTION
**Pull request by londoncalling**
… placeholders, and copyedit fixes

Signed-off-by: Victoria Bialas <victoria.bialas@docker.com>

### CODE REVIEW

**Comment by londoncalling** [comment]
@moxiegirl @FrenchBen I updated the online docs version of the Kitematic Contirbutors' Guide w/some copy-edits, and added the issue # we used for the example (same as the Gist http://git.io/v8d55), can you guys take a quick look at let me know if it's good to go? Thanks! Vicky

File: docs/kitematic/create_pr.md [hunk 1]
```diff
@@ -16,17 +16,15 @@ Rebasing helps verify that your commit is clean, and allows for an easier and fa
 
 Once you've cleaned up your commits:
 
-1. Push your code to your repository under the same name as your local branch.
+1. Push your code to your repository under the same name as your local branch (in the case of this example, 1191-branch).
 
-        $ git push origin xxxx-branch:xxxx-branch
-
-      where _xxxx_ is the number of the issue your worked on.
+        $ git push origin 1191-branch
```
Comment (FrenchBen) [inline docs/kitematic/create_pr.md]: this needs to be `git push origin 1191-branch:1191-branch` in order to create the remote branch with the same name as the local branch. 
The above only works if the remote branch exists and has the same name

File: docs/kitematic/find_issue.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ To find and claim an issue you want to work on:
 
 1. Go to the `kitematic/kitematic` <a href="https://github.com/kitematic/kitematic" target="_blank">repository</a>.
```
Comment (FrenchBen) [inline docs/kitematic/find_issue.md]: Since the docs, we moved `kitematic/kitematic` to `docker/kitematic`

File: docs/kitematic/find_issue.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ To find and claim an issue you want to work on:
 
     The comments on the issues describe the problem and can provide information for a potential solution.
 
-    For this exercise we will select the issue #xxxx Missing container id.
+    For this exercise we will select the issue <a href="https://github.com/kitematic/kitematic/issues/1191" target="_blank">#1191 Missing container id</a>.
```
Comment (FrenchBen) [inline docs/kitematic/find_issue.md]: Same thing here, the proper link is:
`https://github.com/docker/kitematic/issues/1191`

File: docs/kitematic/set_up_dev.md [hunk 4]
```diff
@@ -67,7 +67,7 @@ To fork the master branch of Kitematic:
 
 1. Go to the <a href="https://github.com/docker/kitematic "target="_blank">kitematic/kitematic repository </a>.
```
Comment (FrenchBen) [inline docs/kitematic/set_up_dev.md]: Same as above `docker/kitematic`

File: docs/kitematic/set_up_dev.md [hunk 5]
```diff
@@ -95,9 +95,9 @@ To clone your repository and create a branch for the issue:
 
 4. Create and checkout a branch for the issue you will be working on.
 
-        $ git checkout -b xxxx-branch
+        $ git checkout -b 1191-branch
 
-    where _xxxx_ is the number of the issue previously noted.
+    As previously mentioned, issue <a href="https://github.com/docker/kitematic/issues/1191" target="_blank">#1191</a> is set up as an example to use for this exercise.
```
Comment (FrenchBen) [inline docs/kitematic/set_up_dev.md]: And here again `docker/kitematic`

File: docs/kitematic/set_up_dev.md [hunk 6]
```diff
@@ -67,7 +67,7 @@ To fork the master branch of Kitematic:
 
 1. Go to the <a href="https://github.com/docker/kitematic "target="_blank">kitematic/kitematic repository </a>.
 
-2. Click "Fork" in the upper right corner of the GitHub interface.
+2. Click **Fork** in the upper right corner of the GitHub interface.
 
     GitHub forks the repository to your GitHub account. The original
     `kitematic/kitematic` repository becomes a new fork `YOUR_ACCOUNT/kitematic`
```
Comment (FrenchBen) [inline docs/kitematic/set_up_dev.md]: And here again `docker/kitematic`

**Comment by londoncalling** [comment]
@FrenchBen Thanks! I think I got them all. Let me know, thanks for the quick review.

**Comment by FrenchBen** [comment]
LGTM

**Pull request merged**